source/vocab/things.ttl

```diff
@@ -237,10 +237,11 @@
     owl:equivalentClass bf2:Cartography .
 
 :Dataset a owl:Class;
-    rdfs:label "Dataset"@sv;
-    skos:definition "Data kodad i en definierad struktur. Inkluderar bland annat numeriska data, miljödata. Används av applikationsprogramvara för att beräkna medelvärden, korrelationer, eller för att producera modeller etc. Visas normalt inte i sin råa form."@sv;
-    rdfs:subClassOf :Work;
-    owl:equivalentClass dctype:Dataset, bf2:Dataset, sdo:Dataset .
+    # NOTE: See :Dataset in platform.ttl for more detailed descriptions and vocabulary correlations regarding the notion of Datasets.
+    # rdfs:label "Dataset"@sv;
+    # skos:definition "Data kodad i en definierad struktur. Inkluderar bland annat numeriska data, miljödata. Används av applikationsprogramvara för att beräkna medelvärden, korrelationer, eller för att producera modeller etc. Visas normalt inte i sin råa form."@sv;
+    # rdfs:subClassOf :Work;
+    owl:equivalentClass bf2:Dataset .
```

Comment:
`owl:equivalentClass bf2:Dataset` defined both here and in platform.ttl?

Reply:
Becomes the same statement. I used it as a security measure ;)